### PR TITLE
Re-enable Ubuntu repos in pulp

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

**Comment by Alex-Welsh** [comment]
Waiting on CI before I mark as ready for review

File: etc/kayobe/apt.yml [hunk 1]
```diff
@@ -39,12 +47,25 @@
 # * architecture: whitespace-separated list of architectures that will be used
 #   (optional, default is unset)
 # Default is an empty list.
-#apt_repositories:
+apt_repositories:
+  - url: "{{ stackhpc_repo_ubuntu_focal_url }}"
+    suites: "{{ ansible_facts.distribution_release }} {{ ansible_facts.distribution_release }}-updates {{ ansible_facts.distribution_release }}-backports {{ ansible_facts.distribution_release }}-security"
```
Comment (markgoddard) [inline etc/kayobe/apt.yml]: Wondering why we have security here as well as the security below.

Comment (markgoddard) [reply etc/kayobe/apt.yml]: In commit 31bc7455823f788f1becc5e063914802d68ebbb5 we did not have security here.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/apt.yml (hunk 1 above)
Comment (Alex-Welsh) [reply etc/kayobe/apt.yml]: My mistake, just a transcription error I think

**Review by markgoddard — APPROVED** [review]

**Pull request merged**